ci/unit/test_main.sh

```diff
@@ -27,6 +27,11 @@ else
   BULLSEYE=
 fi
 
+NLT=
```

Comment:
Booleans should always have a `true`/`false` value.

```suggestion
NLT=false
```